app/models/alchemy/page/page_scopes.rb

```diff
@@ -27,7 +27,7 @@ module Page::PageScopes
 
       # All not locked pages
       #
-      scope :not_locked, -> { where(locked: false) }
+      scope :not_locked, -> { where(locked_at: nil, locked_by: nil) }
```

Comment:
Yes, we do make sure these values are set simultaneously in https://github.com/AlchemyCMS/alchemy_cms/blob/2c606393969f1ca056391c82a8855515022bcf5e/app/models/alchemy/page.rb#L324